.gitignore

```diff
@@ -40,4 +40,6 @@
 dependency-reduced-pom.xml
 target/
 tests.log*
-**/nb-configuration*.xml
\ No newline at end of file
+**/nb-configuration*.xml
+/underfs/s3/nbproject/
+/underfs/oss/nbproject/
```

Comment:
Hi @aaudiber !
I am using Netbeans :-) and ... I added Netbeans-specific files to ignore list